pkg/operation/botanist/kubeapiserver_test.go

```diff
@@ -466,21 +462,7 @@ var _ = Describe("KubeAPIServer", func() {
 			})
 
 			Describe("PodSecurity Admission Plugin", func() {
-				var (
-					shootCopy  *gardencorev1beta1.Shoot
-					configData *runtime.RawExtension
-				)
-
-				BeforeEach(func() {
-					shootCopy = botanist.Shoot.GetInfo().DeepCopy()
-					shootCopy.Spec.Kubernetes = gardencorev1beta1.Kubernetes{
-						KubeAPIServer: &gardencorev1beta1.KubeAPIServerConfig{
-							AdmissionPlugins: []gardencorev1beta1.AdmissionPlugin{},
-						},
-						Version: "1.23.0",
-					}
-					botanist.Shoot.KubernetesVersion = semver.MustParse("1.23.0")
-				})
+				var configData *runtime.RawExtension
```

Comment:
This variable can cause some side effects with the way that it is set here https://github.com/gardener/gardener/blob/1da39002ccd90e8b3dc08763dcf41708e48d907f/pkg/operation/botanist/kubeapiserver_test.go#L494 `configData` is never reset for new test runs so it will always point to the data that it was set to from the previous run.